builds/misc/mqtt-perf.yaml

```diff
@@ -17,14 +17,6 @@
 trigger: none
 pr: none
 
-schedules:
```

Comment:
This change doesn't do anything because the pipeline has already been removed. But on the other hand, it doesn't matter! 😉